mobile/lib/app/features/register/view/register_screen.dart

```diff
@@ -0,0 +1,36 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:triggo/app/features/register/bloc/register_bloc.dart';
+import 'package:triggo/app/features/register/view/register_form.dart';
+import 'package:triggo/mediator/authentication.mediator.dart';
+
+class RegisterScreen extends StatelessWidget {
+  const RegisterScreen({super.key});
+
+  static Route<void> route() {
+    return MaterialPageRoute<void>(builder: (_) => const RegisterScreen());
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      appBar: AppBar(
+        backgroundColor: Colors.transparent,
+        leading: IconButton(
+          icon: const Icon(Icons.arrow_back),
+          onPressed: () => Navigator.of(context).pop(),
+        ),
```

Comment:
Maybe I'm wrong, but I think replacing Navigator.pushNamedUntil... with just Navigator.pushNamed() allows you to avoid writing this and handles it by itself

Reply:
No, but this is exactly what here. The user came from the "Welcome" Screen, so He can go back with the "pop". And when he's trying to login or register, He'll be redirected with Navigator.pushNamedUntil() in order to avoid ability to go back to Login/Register Screen with his navigation keys

Reply:
My bad then 👍